js/src/forum/addSettingsItems.js

```diff
@@ -0,0 +1,139 @@
+import { extend } from "flarum/extend";
+
+import SettingsPage from "flarum/components/SettingsPage";
+
+import LoadingIndicator from "flarum/components/LoadingIndicator";
+import Select from "flarum/components/Select";
+import FieldSet from "flarum/components/FieldSet";
+import Switch from "flarum/components/Switch";
+
+import { SetTheme } from "./setSelectedTheme";
+import fixInvalidThemeSetting from "./fixInvalidThemeSetting";
+import GetTheme from "./getTheme";
+import Themes from "./Themes";
+
+const LocalStorageKey = `fofNightMode_deviceTheme`;
```

Comment:
This constant is also defined in `js/src/forum/fixInvalidThemeSetting.js`. Wouldn't it be better to have a single shared constant? Otherwise it's not really a constant... :)

Reply:
Fixed!